domain_source/domain_source.module

```diff
@@ -54,7 +57,11 @@ function domain_source_confirm_fields($entity_type, $bundle) {
   }
 
   // Tell the form system how to behave. Default to radio buttons.
-  if ($display = \Drupal::entityTypeManager()->getStorage('entity_form_display')->load($entity_type . '.' .  $bundle . '.default')) {
+  /** @var $display \Drupal\Core\Entity\Display\EntityDisplayInterface */
+  $display = \Drupal::entityTypeManager()
```

Comment:
Use of `\Drupal::entityTypeManager()` is fine, since that's available in `Drupal.php` we should strive for consistency.

Reply:
?? confused. The code currently uses both `\Drupal::entityTypeManager()` and `\Drupal::service('entity_type.manager')` so is not consistent now.

Reply:
That's because it was written at two different times, and some of it likely before the EntityTypeManager was a service, so it was updated differently. 

We should standardize on `\Drupal::entityTypeManager()` but that can be in a later cleanup.

Reply:
Ok, that's fine.